src/script/components/InputBar/useMessageHandling/useMessageSend/useMessageSend.ts

```diff
@@ -134,31 +137,54 @@ export const useMessageSend = ({
     ],
   );
 
+  const getCellAssets = useCallback((): IAttachment[] => {
+    return files.map(file => {
+      return {
+        cellAsset: {
+          uuid: file.id,
+          contentType: file.type,
+          initialName: file.name,
+          initialSize: file.size,
+        },
+      };
+    });
+  }, [files]);
+
   const sendTextMessage = useCallback(
     (messageText: string, mentions: MentionEntity[]) => {
-      if (messageText.length) {
-        const mentionEntities = mentions.slice(0);
+      const isEmpty = messageText.length === 0;
 
-        void generateQuote().then(quoteEntity => {
-          void messageRepository.sendTextWithLinkPreview(conversation, messageText, mentionEntities, quoteEntity);
-          cancelMessageReply();
-        });
+      if (isEmpty && !cellsEnabled) {
+        return;
       }
+
+      const mentionEntities = mentions.slice(0);
```

Comment:
I don't see any mutation to this variable, is creating a copy necessary?

Reply:
It's an old code